src/components/views/Login.js

```diff
@@ -56,7 +56,13 @@ const Login = () => {
       localStorage.setItem('id', response.data.id)
 
       // Login successfully worked --> navigate to the route /game in the GameRouter
-      history.push(`/game/select/blackCard`);
+      history.push({
+        pathname: `/game/select/blackCard`,
+        state: {
```

Comment:
what does state: do?

Reply:
That passes a state (an object) with history to the next Component.